libraries/ui/src/Card.tsx

```diff
@@ -51,25 +51,27 @@ export const Card: React.FC<CardProps> = ({
 
       <div className="card__content flex flex-col gap-6 w-full flex-1 justify-between">
         <div className="card__text">
-          <h2 className="card__title text-2xl font-semibold text-bluedot-darker mb-1">{title}</h2>
-          {subtitle && (<p className="card__subtitle text-base text-bluedot-black">{subtitle}</p>)}
+          <h2 className="card__title text-2xl font-[650] text-bluedot-darker mb-1">{title}</h2>
+          {subtitle && (<p className="card__subtitle text-sm text-bluedot-black">{subtitle}</p>)}
         </div>
 
-        <div className="card__cta-row flex items-center justify-between mt-auto">
-          {ctaMetadata && (
-            <p className="card__cta-metadata text-left text-xs text-bluedot-black">
-              {ctaMetadata}
-            </p>
+        <div className="card__footer-row flex items-center justify-between mt-auto w-full">
+          {footerContent && (
+            <div className="card__footer-content text-left text-xs text-bluedot-black flex-1">
+              {footerContent}
+            </div>
+          )}
```

Comment:
i'd recommend accepting this as a child instead of a component prop. semantically, i think of this as more of a generic yield than an expected specific value. i'd recommend using `{{children}}` or a [named child](https://dev.to/ncphillips/the-react-named-children-pattern-53g1) `{{children.footer}}`
```suggestion
    {children.footer && (
        <div className="card__footer-row flex items-center justify-between mt-auto w-full">
            <div className="card__footer-content text-left text-xs text-bluedot-black flex-1">
              {children.footer}
            </div>
         </div>
       )}
```
wdyt?